python-lib/ocr/__init__.py

```diff
@@ -36,3 +37,16 @@ def extract_text_ocr(img_bytes, params):
         raise NotImplementedError("OCR engine {} not implemented".format(params[Constants.OCR_ENGINE]))
 
     return img_text
+
+
+def get_multi_page_pdf_page_nb(file_name):
+    if re.match(r"^.*_pdf_page_\d{5}\.jpg$", file_name):
+        return int(re.search(r"^.*_pdf_page_(\d{5})\.jpg$", file_name).group(1))
+    return 1
```

Comment:
You can only use `fullmatch` once here (https://docs.python.org/3/library/re.html#search-vs-match), to match on the full string, no need to call match and search:

```suggestion
    matched = re.fullmatch(r".*_pdf_page_(\d{5})\.jpg", file_name)
    if matched is not None:
        return int(matched.group(1))
    return 1
```